.github/workflows/uberjar.yml

```diff
@@ -2,18 +2,25 @@ name: Uberjar
 
 on:
   push:
-    branches:
-      - '**'
+    branches-ignore:
+      - 'master'
+      - 'release**'
+      - 'backport**'
     paths-ignore:
     - 'docs/**'
     - 'frontend/test/**'
     - 'enterprise/frontend/test/**'
+    - ".**"
+    - "test*"
 
-jobs:
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
 
+jobs:
   build:
     runs-on: ubuntu-20.04
-    timeout-minutes: 60
+    timeout-minutes: 40
```

Comment:
Accidental change?

Reply:
yup, but this shouldn't take more than 30 mins, so 40 is more than enough, so I'll keep this value in 40

Reply:
Something it does, particularly when `deps.edn` and `package.json` are both updated, hence also both the M2 and yarn cache aren't useful. Therefore, fetching fresh dependencies from Maven and npmjs can affect the duration (we're subjected to external network condition at that very moment).